app/src/app/constants/configuration.ts

```diff
@@ -1,14 +1,1935 @@
+    layout: {
+      "text-font": ["Barlow Medium"],
+      "text-field": ["get", "name"],
+      "text-size": ["interpolate", ["linear"], ["zoom"], 3, 0, 6, 12, 10, 12],
+      "text-letter-spacing": 0.1,
+      "text-max-width": 9,
+    },
+    paint: {
+      "text-color": "#adadad",
+    },
+  },
+  {
+    id: "roads_labels_major",
+    type: "symbol",
+    source: "protomaps",
+    "source-layer": "roads",
+    minzoom: 11,
+    filter: [
+      "any",
+      ["in", "pmap:kind", "highway", "major_road", "medium_road"],
+    ],
+    layout: {
+      "symbol-sort-key": ["get", "pmap:min_zoom"],
+      "symbol-placement": "line",
+      "text-font": ["Barlow Regular"],
+      "text-field": ["get", "name"],
+      "text-size": 12,
+    },
+    paint: {
+      "text-color": "#999999",
+      "text-halo-color": "#ffffff",
+      "text-halo-width": 2,
+    },
+  },
+  {
+    id: "places_subplace",
+    type: "symbol",
+    source: "protomaps",
+    "source-layer": "places",
+    filter: ["==", "pmap:kind", "neighbourhood"],
+    layout: {
+      "symbol-sort-key": ["get", "pmap:min_zoom"],
+      "text-field": "{name}",
+      "text-font": ["Barlow Regular"],
+      "text-max-width": 7,
+      "text-letter-spacing": 0.1,
+      "text-padding": [
+        "interpolate",
+        ["linear"],
+        ["zoom"],
+        5,
+        2,
+        8,
+        4,
+        12,
+        18,
+        15,
+        20,
+      ],
+      "text-size": [
+        "interpolate",
+        ["exponential", 1.2],
+        ["zoom"],
+        11,
+        8,
+        14,
+        14,
+        18,
+        24,
+      ],
+      "text-transform": "uppercase",
+    },
+    paint: {
+      "text-color": "#8f8f8f",
+      "text-halo-color": "#ffffff",
+      "text-halo-width": 1.5,
+    },
+  },
+  {
+    id: "counties_boundary",
+    type: "line",
+    source: "counties",
+    "source-layer": "tl_2023_us_county",
+    paint: {
+      "line-color": "#555",
+      "line-width": [
+        "interpolate",
+        ["exponential", 1.6],
+        ["zoom"],
+        6,
+        0,
+        9,
+        0.75,
+        18,
+        1,
+      ],
+    },
+  },
+  {
+    id: "counties_labels",
+    type: "symbol",
+    source: "counties",
+    "source-layer": "tl_2023_us_county_label",
+    minzoom: 6,
+    layout: {
+      "text-field": ["get", "NAME"],
+      "text-font": ["Barlow Regular"],
+      "text-size": 10,
+      "text-transform": "uppercase",
+      "text-letter-spacing": 0.1,
+      "text-max-width": 9,
+      "text-padding": ["interpolate", ["linear"], ["zoom"], 5, 3, 8, 7, 12, 11],
+    },
+    paint: {
+      "text-color": "#666",
+      "text-halo-color": "#ffffff",
+      "text-halo-width": 2,
+    },
+  },
+  {
+    id: "places_locality",
+    type: "symbol",
+    source: "protomaps",
+    "source-layer": "places",
+    filter: ["==", "pmap:kind", "locality"],
+    layout: {
+      "icon-image": ["step", ["zoom"], "townspot", 8, ""],
```

Comment:
this icon is not loaded or referenced properly:
<img width="920" alt="image" src="https://github.com/user-attachments/assets/1bf6c310-1e74-4d99-9328-de9f7403b7b9">